src/main/java/com/ldtteam/blockui/controls/ButtonImage.java

```diff
@@ -198,13 +198,16 @@ private void loadImageDisabledDimensions(final ResourceLocation rl)
      */
     public void setImage(final ResourceLocation loc, final int offsetX, final int offsetY, final int w, final int h)
     {
+        if (!Objects.equals(loc, image))
+        {
+            loadImageDimensions(loc);
+        }
+
         image = loc;
         imageOffsetX = offsetX;
         imageOffsetY = offsetY;
```

Comment:
should that if be after the value setting like how it was before?

Reply:
doesnt matter